### ISSUE TITLE
Pip install failing in  py3k

### ISSUE THREAD

**Issue by rrshaban** [mdp-toolkit/mdp-toolkit#14]
```
>>> import mdp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/razi/git/yelp/src/mdp/build/py3k/mdp/__init__.py", line 114, in <module>
    from . import configuration
  File "/Users/razi/git/yelp/src/mdp/build/py3k/mdp/configuration.py", line 186
    except ImportError, exc:
                      ^
SyntaxError: invalid syntax
```

**Comment by rrshaban** [comment]
When trying to build it manually, it installs without warning, but then when I try to import:
```
>>> import mdp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/razi/git/yelp/mdp-toolkit/mdp/__init__.py", line 114, in <module>
    import configuration
ImportError: No module named 'configuration'
```

**Comment by otizonaizit** [comment]
Is this with Python 3 or Python 2? Can you post the exact steps you used to install with pip (with complete output from pip) and for the manual installation (with complete log)? That would help understanding what the issue is…

**Comment by rrshaban** [comment]
```
$ pip3 install mdp
Downloading/unpacking mdp
  Could not find any downloads that satisfy the requirement mdp
  Some externally hosted files were ignored (use --allow-external mdp to allow).
Cleaning up...
No distributions at all found for mdp
Storing debug log for failure in /Users/razi/.pip/pip.log
```
pip.log:
```
------------------------------------------------------------
/usr/local/bin/pip3 run on Tue Apr 28 10:24:55 2015
Downloading/unpacking mdp
  Getting page https://pypi.python.org/simple/mdp/
  URLs to search for versions for mdp:
  * https://pypi.python.org/simple/mdp/
  Analyzing links from page https://pypi.python.org/simple/mdp/
    Skipping link http://mdp-toolkit.sourceforge.net (from https://pypi.python.org/simple/mdp/); not a file
    Skipping link http://sourceforge.net/project/showfiles.php?group_id=116959 (from https://pypi.python.org/simple/mdp/); unknown archive format: .php
    Skipping link http://sourceforge.net/projects/mdp-toolkit/files (from https://pypi.python.org/simple/mdp/); not a file
    Skipping link http://sourceforge.net/projects/mdp-toolkit/files/mdp-toolkit/3.0/MDP-3.0.tar.gz/download (from https://pypi.python.org/simple/mdp/); not a file
    Skipping http://sourceforge.net/projects/mdp-toolkit/files/mdp-toolkit/3.3/MDP-3.3.tar.gz (from https://pypi.python.org/simple/mdp/) because it is externally hosted.
  Could not find any downloads that satisfy the requirement mdp
  Some externally hosted files were ignored (use --allow-external mdp to allow).
Cleaning up...
  Removing temporary dir /private/var/folders/wd/w3dy49v152b3lxcsvqgvnj280000gn/T/pip_build_razi...
No distributions at all found for mdp
Exception information:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/commands/install.py", line 278, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/req.py", line 1177, in prepare_files
    url = finder.find_requirement(req_to_install, upgrade=self.upgrade)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/index.py", line 277, in find_requirement
    raise DistributionNotFound('No distributions at all found for %s' % req)
pip.exceptions.DistributionNotFound: No distributions at all found for mdp
```

**Comment by rrshaban** [comment]
And yes, the second issue, the manual build resulting in a `No module named 'configuration'` error seemed to be caused by the Python3to2 convertor, which ran before installing. 

I've been able to pip install on Python2 just fine.

**Comment by otizonaizit** [comment]
To install on python3 you should probably to what pip3 is suggesting you in the error message:

```Some externally hosted files were ignored (use --allow-external mdp to allow).```

So, does it work if your try:

```pip3 install --allow-external mdp```

If this is not enough, you can try:

```pip3 install --allow-external --allow-unverified mdp```

New versions of `pip` do not like externally hosted files...

**Comment by otizonaizit** [comment]
fixed in 21d6ae8c2e8e63ff08cbf7d07249885ff6459285
